src/cim/components/ClusterDeployment/ACMClusterDeploymentNetworkingStep.tsx

```diff
@@ -13,12 +13,13 @@ import {
   ClusterDeploymentNetworkingValues,
 } from './types';
 
-type ACMClusterDeploymentNetworkingStepProps = ClusterDeploymentHostsTablePropsActions & {
+type ACMClusterDeploymentNetworkingStepProps = {
   clusterDeployment: ClusterDeploymentK8sResource;
   agentClusterInstall: AgentClusterInstallK8sResource;
   pullSecretSet: boolean;
   agents: AgentK8sResource[];
   onValuesChanged: (values: ClusterDeploymentNetworkingValues) => void;
+  hostActions: ClusterDeploymentHostsTablePropsActions;
```

Comment:
@rawagner This is braking change for ACM. To fix, just wrap those `onSomething()` callbacks into `hostActions` object